src/controllers/wallet/p2sh/tx-proposal.controller.js

```diff
@@ -33,6 +34,7 @@ async function buildTxProposal(req, res) {
     }
   }
   try {
+    storage.setStore(req.wallet.store);
```

Comment:
I would just add a comment here saying that this should be in the lib and that we should move it there as soon as we have a more complete method to do it in the wallet class.

Reply:
Done